### PR TITLE
Add instructions for validating the DNS server.

### PR DESCRIPTION
**Pull request by brendandburns**
@juliaferraioli @kprobst @erictune 

Closes https://github.com/GoogleCloudPlatform/kubernetes/issues/9221

### CODE REVIEW

**Comment by ArtfulCoder** [comment]
LGTM

File: cluster/addons/dns/README.md [hunk 1]
```diff
@@ -95,6 +95,63 @@ Salt.  You will need to replace the `{{ <param> }}` blocks with your own values
 for the config variables mentioned above.  Other than the templating, these are
 normal kubernetes objects, and can be instantiated with `kubectl create`.
 
+## How do I test if it is working?
+First deploy DNS as described above.
+
+The create a simple Pod to use as a test environment.  Create a file named busybox.yaml with the
```
Comment (juliaferraioli) [inline cluster/addons/dns/README.md]: s/The/Then

**Comment by juliaferraioli** [comment]
Consider reformatting in an ordered list, as these instructions are order dependent.

File: cluster/addons/dns/README.md (hunk 1 above)
Comment (brendandburns) [inline cluster/addons/dns/README.md]: done.

**Comment by brendandburns** [comment]
Comments addressed, ptal.

Thanks!
--brendan

**Comment by juliaferraioli** [comment]
LGTM

**Comment by brendandburns** [comment]
two LGTMs from others, so I added the LGTM label.

**Pull request merged**